actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -364,6 +375,16 @@ def multiple_sources_tag_builder(type, sources)
           end
         end
 
+        def resolve_image_source(source, skip_pipeline)
+          if source.is_a?(Symbol) || source.is_a?(String)
```

Comment:
I think this should be something `polymorpic_url` should do. Be cause if the object is not a model or has a `resolve` we need to tell the users what they can do.